### PR TITLE
Adding info for the patch release management team

### PR DESCRIPTION
**Pull request by claurence**
Per a thread in slack last week adding a breadcrumb to the patch release management team for 1.15.0 so folks know who to reach out to

### CODE REVIEW

**Comment by claurence** [comment]
cc @justaugustus @tpepper  nothing urgent - trying to add info for 1.15.0 release team on patch management for the release since I saw a question come in since we shipped last week on who to direct patch info to

File: releases/release-1.15/release_team.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 | Docs | Barnabas Makonda ([@MAKOSCAFEE](https://github.com/MAKOSCAFEE) / Slack: `@barnie`) | Tunde Oladipupo ([@simplytunde](https://github.com/simplytunde) / Slack: `@simplytunde`), Damini Satya Kammakomati ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini satya`), Christian Hernandez ([@christianh814](https://github.com/christianh814) / Slack: `@christianh814`) |
 | Release Notes | Lindsey Tulloch ([@onyiny-ang](https://github.com/onyiny-ang) / Slack: `@onyiny-ang`) | Jeff Sica ([@jeefy](https://github.com/jeefy) / Slack: `@jeefy`), Nick Chase ([@NickChase](https://github.com/NickChase) / Slack: `@nickchase`), Sascha Grunert ([saschagrunert](https://github.com/saschagrunert) / Slack: `@sgrunert`), Alok Patra ([alokpatra](https://github.com/alokpatra) / `@Alok Patra`)|
 | Communications | Jorge Castro ([@castrojo](https://github.com/castrojo)) | Taylor Dolezal ([@onlydole](https://github.com/onlydole)), David McKay ([@rawkode](https://github.com/rawkode))|
-
+| Patch Release Management | [Patch Release Team](https://github.com/orgs/kubernetes/teams/patch-release-team) |
```
Comment (justaugustus) [inline releases/release-1.15/release_team.md]:
```suggestion
Review the [Patch Releases page](/releases/patch-releases.md) for up-to-date contact information and the schedule for 1.15 patch releases.
```

**Comment by claurence** [comment]
thanks!  added suggestion!

**Comment by idealhack** [comment]
GitHub treated it as a cell of the table, I fixed it in https://github.com/kubernetes/sig-release/pull/702/commits/3e449d27e3ef1e5f1f0026117fcdde6867329c8f

**Pull request merged**